### DISCUSSION TITLE
Best-practices for safely removing FluxCD

### DISCUSSION THREAD

**Discussion by lrdplopes** [fluxcd/flux2#5126] [Q&A]
Hey, community.

We are planning to eliminate FluxCD from our infrastructure. Currently, Flux manages several Kubernetes resources, and we want to ensure these resources remain intact and operational during and after the transition.

Could you please advise on the best practices for safely removing FluxCD while ensuring all resources managed by it are retained?
Thank you so much.

**Comment by stefanprodan** [comment]
Check out the uninstall command: https://fluxcd.io/flux/installation/uninstall/

**Reply by lrdplopes** [reply]
Thank you very much for the quick answer, @stefanprodan.
So, does this mean that uninstalling Flux through `CLI` will not affect any resources that Flux manages?

**Reply by stefanprodan** [reply]
From the docs I linked above:

> The uninstallation procedure removes only the Flux components, without touching reconciled namespaces, tenants, cluster addons, workloads, Helm releases, etc.
